spec/controllers/evaluation_controller_spec.rb

```diff
@@ -150,5 +150,36 @@
     end
 
   end
+  
+  describe "POST #upload" do
+    it "accepts .xlsx files for uploading" do
+      @file = fixture_file_upload('/StatisticsReport.xlsx', 'application/vnd.ms-excel')
+      post :upload, :data_file => @file
+      expect(response).to redirect_to("/evaluation")
+      expect(response.body).to have_content("Term")
+    end
```

Comment:
Please add a new line separating individual examples. See other `describe` blocks in this file for examples.